pkg/gui/services/custom_commands/handler_creator.go

```diff
@@ -191,7 +193,11 @@ func (self *HandlerCreator) finalHandler(customCommand config.CustomCommand, ses
 		if err != nil {
 			return self.c.Error(err)
 		}
+		if strings.TrimSpace(output) == "" {
+			output = self.c.Tr.EmptyOutput
+		}
 		if customCommand.ShowOutput {
+			self.c.Refresh(types.RefreshOptions{})
```

Comment:
Shouldn't this be done _after_ invoking the `cmdStr` in case it changes something?

Reply:
TBH I'm not really sure what refresh does, as this is my first time working with this codebase. Should I change it to this?
```go
self.c.Alert(cmdStr, output)
return self.c.Refresh(types.RefreshOptions{})
```

Reply:
The docs say:
```
we call this when we want to refetch some models and render the result. Internally calls PostRefreshUpdate
```

Since we don't know what the user might have under a custom command, maybe creating a new branch and committing something there, we need to refresh.

> self.c.Alert(cmdStr, output)
return self.c.Refresh(types.RefreshOptions{})

This seems fine, just be sure to check the return status of `Alert` :)

Reply:
Ah, I see. That makes sense, thanks! I'll make the change now